app/graphql/mutations/users/update_user_optin_status.rb

```diff
@@ -0,0 +1,15 @@
+module Mutations
+  module Users
+    class UpdateUserOptinStatus < ::Mutations::BaseMutation
+      argument :id, ID, required: true
+
+      type Types::UserType
+
+      def resolve(attributes)
+        user = User.where(id: attributes[:id]).first
+        !user.rock_opt_in ? user.update(rock_opt_in: true) : user.update(rock_opt_in: false)
```

Comment:
Nice job! I always forget about the ":" as an option for one line conditional statements!